### DISCUSSION TITLE
OAuth Callback URL Configuration?

### DISCUSSION THREAD

**Discussion by sir-farfan** [hoppscotch/hoppscotch#4522] [Q&A] [answered]
[You must configure this URL as a valid redirect URI](https://docs.hoppscotch.io/documentation/features/authorization#oauth-callback-url-configuration)

Hey there guys
Currently I'm trying from the browser with the plugin, I'm new to this.

I've chosen Authorization -> Oauth 2.0 -> Code

But, where's the Callback URL config box?

In Postman it's right there with the other required parameters

**Comment by SanskritiHarmukh** [comment, marked as answer]
Hi there! When using OAuth 2.0 with Hoppscotch Cloud, the callback URL is fixed to `https://hoppscotch.io/oauth`. However, for self-hosted editions of Hoppscotch, the callback URL should be set to `<your-domain>/oauth`. You can refer to our [documentation to learn more](https://docs.hoppscotch.io/documentation/features/authorization#oauth-callback-url-configuration).

**Reply by rvanlaak** [reply]
Am still getting an error on returning to the in-browser Hoppscotch cloud version. This while the same configuration on Postman does work correctly. Anything more I can do to debug this?

**Reply by marcel-hoell** [reply]
Same for me...any news on this?

**Reply by wapmorgan** [reply]
wait for update that will work as in postman

**Reply by wapmorgan** [reply]
Any updates on this?

**Reply by silex-fabrice** [reply]
Hello, the documentation is wrong, the desktop app, always send a auth url callback with http.
So I think an issue need to be open to fix it.
